pyproject.toml

```diff
@@ -176,6 +176,8 @@ select = [
   "F",   # pyflakes
   "PL",  # pylint
   "NPY", # numpy specific rules
+  "C4", # flake8-comprehensions
+  "ASYNC", # flake8-async
```

Comment:
Here we should decide on aligning hashes, or switch to ruff/black standard which is two spaces before the hash..